tests/integration/servicecatalog/service_test.go

```diff
@@ -36,6 +36,40 @@ var _ = Describe("odoServiceE2e", func() {
 		})
 	})
 
+	Context("create service with Env non-interactively", func() {
+		JustBeforeEach(func() {
+			project = helper.CreateRandProject()
+			context = helper.CreateNewContext()
+			app = helper.RandString(7)
+			originalDir = helper.Getwd()
+			helper.Chdir(context)
+		})
+		JustAfterEach(func() {
+			helper.DeleteProject(project)
+			helper.DeleteDir(context)
+			helper.Chdir(originalDir)
+		})
+
+		It("should be able to create postgresql with env", func() {
+			oc.ImportJavaIsToNspace(project)
+			helper.CopyExample(filepath.Join("source", "openjdk-sb-postgresql"), context)
+
+			// Local config needs to be present in order to create service https://github.com/openshift/odo/issues/1602
+			helper.CmdShouldPass("odo", "create", "java", "sb-app", "--project", project)
```

Comment:
We don't need a context anymore to create a service since #2001 got merged. So I think we can safely remove above three lines of code.

Reply:
resolved